src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -8,32 +8,24 @@ export const RequestRow = ({
   currentTime,
   isFirstInFuture,
   isInPast,
+  isSelected,
   onClick,
   onSeek,
   row,
 }: {
   currentTime: number;
   isFirstInFuture: boolean;
   isInPast: boolean;
+  isSelected: boolean;
   onClick: (row: RequestSummary) => void;
   onSeek: (row: RequestSummary) => void;
   row: Row<RequestSummary>;
 }) => {
-  // let currentRow = false;
-  // Did we just pass the row boundary that contains the current time?
-  // If so, let's render this row as the "current" row and all rows
-  // after it as future rows.
-  // if (
-  //   selectedRequest?.id === row.original.id ||
-  //   (!selectedRequest && renderingRequestsInThePast && currentTime <= row.original.point.time)
-  // ) {
-  //   renderingRequestsInThePast = false;
-  //   currentRow = true;
-  // }
```

Comment:
should we delete?

Reply:
Well, we already have this feature back in, so I figure we can safely delete the old implementation.